packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx

```diff
@@ -13,7 +13,11 @@ import { SignInExperienceForm } from '../types';
 import ManageLanguageModal from './ManageLanguageModal';
 import * as styles from './index.module.scss';
 
-const LanguagesForm = () => {
+type Props = {
+  isManageLanguageVisible?: boolean;
+};
+
+const LanguagesForm = ({ isManageLanguageVisible = true }: Props) => {
```

Comment:
nit: We usually set this type of flag to `false` by default. Then you only need to add `isLanguageManagerVisible` in the "others" tab to show it

Reply:
updated.